### ISSUE TITLE
Add options for data upload

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/Goliath#17]
Upload data as textfile (drag and drop, only tsv or txt files, one ID per line, no additional columns) or copy paste. This I see as an option we do want for initial release...

**Comment by Chris-Evelo** [comment]
Agree, in addition, the conditions for supported data upload formats should be such that what typically works on DAVID should also work on Goliath.
